documentation/source/index.rst

```diff
@@ -1,3 +1,45 @@
-:orphan:
+===============
+vulkan-renderer
+===============
+
+|language| |platforms| |github actions| |readthedocs| |license|
+
+|issues| |last commit| |code size| |contributors| |discord| |downloads|
+
+**Inexor is a MIT-licensed open-source project which develops a new 3D octree game engine by combining modern C++ with Vulkan API.**
+
+.. image:: /images/banner.jpg
+
+
```

Comment:
```suggestion
.. include:: /faq/main.rst

.. toctree::
    :maxdepth: 2
    :hidden:

    contact/main
    faq/main
    code/main
    development/main
    contributing/main
    changelog/main
    links/main
    license/main

```

Reply:
Also i would not put the links onto the main page.

Reply:
You are right. I will remove it from the main page. It's still available in the menu under "helpful links".

Reply:
Done.